cirrus/server/tests/conftest.py

```diff
@@ -51,3 +53,101 @@ def fml():
 @fixture
 def exception():
     return Exception("some error")
+
+
+@fixture(name="reset_glean", scope="function", autouse=True)
+def fixture_reset_glean():
+    testing.reset_glean(application_id="cirrus-test", application_version="0.1.0")
+
+
+@fixture
+def get_telemetry():
+    return initialize_glean()
+
+
+@fixture
+def create_recipe():
+    def _create(
+        slug="test",
+        app_id="org.mozilla.test",
+        app_name="test_app",
+        channel="release",
+        feature="test-feature",
+        is_rollout=False,
+        targeting="true",
+        bucket_count=10000,
+        bucket_start=0,
+        bucket_total=10000,
+        bucket_randomization_unit="user_id",
+        is_enrollment_paused=False,
+    ):
+        recipe = {
+            "slug": slug,
+            "appId": app_id,
+            "appName": app_name,
+            "channel": channel,
+            "endDate": None,
+            "locales": None,
+            "branches": [
+                {
+                    "slug": "control",
+                    "ratio": 1,
+                    "feature": {
+                        "value": {"enabled": True},
+                        "featureId": feature,
+                    },
+                    "features": [
+                        {
+                            "value": {"enabled": True},
+                            "featureId": feature,
+                        }
+                    ],
+                }
+            ],
+            "outcomes": [],
+            "arguments": {},
+            "isRollout": is_rollout,
+            "probeSets": [],
+            "startDate": "2023-07-05",
+            "targeting": targeting,
+            "featureIds": [feature],
+            "application": app_id,
+            "bucketConfig": {
+                "count": bucket_count,
+                "start": bucket_start,
+                "total": bucket_total,
+                "namespace": feature + "-" + slug,
+                "randomizationUnit": bucket_randomization_unit,
+            },
+            "localizations": None,
+            "schemaVersion": "1.12.0",
+            "userFacingName": "",
+            "referenceBranch": "control",
+            "proposedDuration": 28,
+            "enrollmentEndDate": "2023-07-12",
+            "isEnrollmentPaused": is_enrollment_paused,
+            "proposedEnrollment": 7,
+            "userFacingDescription": "",
+            "featureValidationOptOut": False,
+            "id": slug,
+            "last_modified": 1689000336881,
+        }
+        check_schema("experiments/NimbusExperiment", recipe)
```

Comment:
anything that's made from the `create_recipe` fixture will be validated!

Reply:
Excellent 👍